coding-challenges.js

```diff
@@ -52,7 +88,11 @@ var testString2 = "jabberwocky"
 // --------------------5) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator.
 
 
-
+allOriginal = (array1, array2) => {
+    newArr = array1.concat(array2)
+    newArr = [...new Set(newArr)]
+    return newArr
+}
```

Comment:
good use of Set. obv a very efficient solution. just be prepared for an interviewer to ask you to write out the long way. i always suggest considering an optimized solution but for the sake of practice, i would put a "naive" solution (meaning, less efficient or possibly your initial approach), and an "optimized" solution in your assessment. this practice will have a real-world application as well as it is _extremely_ helpful/impressive in an interview setting. showing prospective employers how you approach a problem and how you go about refactoring is very valuable.